### PR TITLE
feat: add assetlinks file for android deep links

### PR DESCRIPTION
**Pull request by admbtlr**
This file is needed in order to make artsy.net links open in the Android app

### CODE REVIEW

**Comment by lordkiz** [comment]
LGTM.

**Review by lordkiz — APPROVED** [review]

File: src/desktop/public/.well-known/assetlinks.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[{
+  "relation": ["delegate_permission/common.handle_all_urls"],
+  "target": {
+    "namespace": "android_app",
+    "package_name": "net.artsy.app",
+    "sha256_cert_fingerprints":
+    ["FA:C6:17:45:DC:09:03:78:6F:B9:ED:E6:2A:96:2B:39:9F:73:48:F0:BB:6F:89:9B:83:32:66:75:91:03:3B:9C"]
```
Comment (pvinis) [inline src/desktop/public/.well-known/assetlinks.json:7]: quick question, where are all these coming from?

File: src/desktop/public/.well-known/assetlinks.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[{
+  "relation": ["delegate_permission/common.handle_all_urls"],
```
Comment (pvinis) [inline src/desktop/public/.well-known/assetlinks.json:2]: is that the thing that tells the phone to look in the app manifest?

File: src/desktop/public/.well-known/assetlinks.json (hunk 1 above)
Comment (lordkiz) [reply src/desktop/public/.well-known/assetlinks.json:7]: @pvinis I think the sha256 fingerprint there is a debug one generated by android studio for this project. Before releasing this for production use, there is need to add the sha256 fingerprint from google play store.

**Pull request merged**